cartographer/mapping_2d/local_trajectory_builder.h

```diff
@@ -84,10 +84,11 @@ class LocalTrajectoryBuilder {
       const Eigen::Quaterniond& gravity_alignment);
 
   // Scan matches 'gravity_aligned_range_data' and fill in the
-  // 'pose_observation' with the result.
-  void ScanMatch(common::Time time, const transform::Rigid2d& pose_prediction,
-                 const sensor::RangeData& gravity_aligned_range_data,
-                 transform::Rigid2d* pose_observation);
+  // 'pose_observation' with the result. Return the observed pose, or nullptr on
```

Comment:
Replace "and fill in ... with the result." by "and returns"

Reply:
Done.